File: app/src/main/java/com/students/studensattendance/Student.java
package com.students.studensattendance;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.students.studensattendance.model.StudentDetail;

import java.util.List;

public class Student {
   @SerializedName("Student Details")
   @Expose
   private List<StudentInfo> studentDetails = null;

   public List<StudentInfo> getStudentDetails() {
      return studentDetails;
   }

   public void setStudentDetails(List<StudentInfo> studentDetails) {
      this.studentDetails = studentDetails;
   }
}

File: app/src/main/java/com/students/studensattendance/model/Student.kt
package com.students.studensattendance.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Student(
   // var StudentDetail : ArrayList<StudentDetail>
    @SerializedName("Student Details")
    val results:List<StudentDetail>
   // var StudentDetail : ArrayList<StudentDetail> = arrayListOf()

)